examples/temperature_example/temperature_example.cc

```diff
@@ -61,26 +61,35 @@ int main(int argc, char *argv[]) {
   auto spatial_cov = angular_exp * radial_sqr_exp;
 
   auto covariance = elevation_scaled_mean + noise + spatial_cov;
-  auto model = gp_from_covariance<Station>(covariance);
-
-  // These parameters are that came from tuning the model to the leave
-  // one out negative log likelihood.
+  auto model = gp_from_covariance(covariance);
+
+  model.set_param("sigma_exponential", {1., std::make_shared<FixedPrior>()});
+
+  //   These parameters are that came from tuning the model to the leave
+  //   one out negative log likelihood. which can be done like this:
+  //
+  //      albatross::LeaveOneOutLikelihood<MarginalDistribution> loo_nll;
+  //      auto tuner = get_tuner(model, loo_nll, data);
+  //      tuner.initialize_optimizer(nlopt::LN_NELDERMEAD);
+  //      const auto params = tuner.tune();
+  //      model.set_params(params);
+  //
   model.set_param_values({
```

Comment:
Why are the parameters changing for the temperture example? Shouldn't functionality be unchanged?
Is this related to the sign flip in the loglikilihood metric?

Reply:
Yeah, I started looking closer at the parameters and decided something was strange.  The `squared_exponential_length_scale` for example is related to a change in altitude and with it as low as it was it would have been completely decorrelating statoins at just slightly different altitudes.  When I changed the optimization algorithm (and fixed `sigma_exponential` which was superfluous) it converged to these new params which seem more realistic to me.